cmd/kpod/spec.go

```diff
@@ -372,24 +382,25 @@ func (c *createConfig) GetVolumeMounts() ([]spec.Mount, error) {
 			options = strings.Split(spliti[2], ",")
 		}
 		options = append(options, "rbind")
-		// var foundrw, foundro,
-		var foundz, foundZ bool
+		var foundrw, foundro, foundz, foundZ bool
+		var rootProp string
 		for _, opt := range options {
 			switch opt {
-			// case "rw":
-			// 	foundrw = true
-			// case "ro":
-			// 	foundro = true
+			case "rw":
+				foundrw = true
+			case "ro":
+				foundro = true
 			case "z":
 				foundz = true
 			case "Z":
 				foundZ = true
+			case "private", "rprivate", "slave", "rslave", "shared", "rshared":
```

Comment:
I thought we were verifying that the user did not add these more then once?

Reply:
It is being verified in create_cli.go. This just adds "private" (default) to the list of options if no root propagation option is given.